src/components/Rating/Rating.jsx

```diff
@@ -0,0 +1,61 @@
+function Rating(props) {
+    const {children} = props
+
+    const roundRating = (rating) => {
+        switch(Math.round(rating)) {
+            case 1:
+                return '★☆☆☆☆'
+                break;
+            case 2:
+                return '★★☆☆☆';
+                break;
+            case 3:
+                return '★★★☆☆';
+                break;
+            case 4:
+                return '★★★★☆';
+                break;
+            case 5:
+                return '★★★★★';
+                break;
+            default: 
+                return '☆☆☆☆☆'
+                break;
+        }
+         
```

Comment:
Here, this works but it's very manual. You could do it in different ways such as a loop or using string.repeat()